run_old_versions.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: install and run heat eol tests from upstream
+  hosts: undercloud
+  vars:
+    ansible_python_interpreter: "/home/stack/ir_venv/bin/python"
+  roles:
+    - testconfig
+  tasks:
+    - name: install git repo with tags
+      git:
+        repo: 'https://github.com/openstack/heat.git'
+        dest: /home/stack/heat
+        version: tags/mitaka-eol
+        clone: no
+        update: no
+
+
```

Comment:
Add include of pip install to ensure it is installed when you creating a venv.

Reply:
done, please review